specifications/sep-payment-channel-mechanism.md

```diff
@@ -196,12 +197,22 @@ multisig accounts. F has source account E, and sequence number set to s_i.
   - Operations sponsored by R:
     - One `BEGIN_SPONSORING_FUTURE_RESERVES` operation that specifies reserve
     account R as a sponsor of future reserves.
+    - One or more `CHANGE_TRUST` operations configuring trustlines on ER.
     - One `SET_OPTIONS` operations adjusting escrow account ER's thresholds such
     that R and I's signers must both sign.
     - One or more `SET_OPTIONS` operations adding R's signers to EI.
     - One `END_SPONSORING_FUTURE_RESERVES` operation that stops R sponsoring
     future reserves of subsequent operations.
   
+  The escrow accounts EI and ER will likely have all the necessary trustlines
+  before the formation transaction is built. This means the `CHANGE_TRUST`
+  operations will likely be no-ops. The `CHANGE_TRUST` operations must be
```

Comment:
does no-op just mean an operation that has no effect? If so makes sense, just double checking

Reply:
Yup.

Reply:
I don't mind adding more length language to state that, but I noticed CAP-21 uses the same terminology, so I copied it.